test/TestUtilities/Test.Utility/SimpleTestSetup/SimpleTestPackageContext.cs

```diff
@@ -70,10 +71,18 @@ public SimpleTestPackageContext()
         public string RuntimeJson { get; set; }
 
         public bool IsSymbolPackage { get; set; }
+        public bool IsSnupkgPackage { get; set; } = false;
```

Comment:
You don't need to explicitly set a default.

Reply:
This is a new property, and none of the existing code knows about its existence. Rather than modify all of those places, I prefer simply defaulting it.

Reply:
bool properties, by default, will default to false. that may be @nkolev92's point?

Reply:
Sure, but since it's important to the behavior, it should be there. Also if that's the case then this comment should be a "nit:"

Reply:
Discussed offline. 
No need for default values.